### ISSUE TITLE
High contrast: Diff view needs high contrast love

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#3291]
#3248 
* High contrast theme, make some changes open diff view in Git
* I think we should use more colors here to make it clearer to the user what lines were deleted / added 

![screen shot 2016-02-23 at 11 11 27](https://cloud.githubusercontent.com/assets/1926584/13248393/4dddd984-da1e-11e5-98df-8303d463dbfe.png)

**Comment by bgashler1** [comment]
This is what my commit will look like.  We cannot use background colors due to contrast issues.  So, I've changed the borders to be colorful.  Note that that the purplish-red against the green is intentional as I wanted to make it visible for color blind users (both deuteranomaly and protanomaly color blindness types).

![image](https://cloud.githubusercontent.com/assets/11839736/13276920/fbcb3064-da78-11e5-84ea-bafecd3357ed.png)

**Comment by joaomoreno** [comment]
@bgashler1 either push your changes to our master branch or don't close your items

**Comment by joaomoreno** [comment]
:+1:
